samples/apps/copilot-chat-app/webapp/src/libs/services/ChatService.ts

```diff
@@ -9,22 +9,18 @@ import { BaseService } from './BaseService';
 
 export class ChatService extends BaseService {
     public createChatAsync = async (
-        userId: string,
-        userName: string,
         title: string,
         accessToken: string,
     ): Promise<IChatSession> => {
         const body = {
-            userId: userId,
-            userName: userName,
-            title: title,
+            title,
         };
 
         const result = await this.getResponseAsync<IChatSession>(
             {
-                commandPath: 'chatSession/create',
+                commandPath: 'chatSessions',
```

Comment:
Chat sessions urls were altered to more closely follow microsoft's api design standards.